VelocitySDK/Support/VLTProtobufHelper.m

```diff
@@ -54,7 +51,6 @@ + (nonnull VLTPBDetectMotionRequest *)detectMotionRequestFromDatas:(nonnull NSAr
     motionRequest.userId                    = userId;
     motionRequest.modelNameArray            = [modelNames mutableCopy];
     motionRequest.sensorsArray              = sensors;
-    motionRequest.sequenceIndex             = seqIndex;
```

Comment:
Because it is a required field: https://github.com/velocity-inc/velocity-protobuf/blob/master/definitions/velocity.proto#L40

Probably set it to `0`.